src/course-page-resources/index.scss

```diff
@@ -0,0 +1,3 @@
+.course-page-config-card {
+  flex-basis: 100%;
```

Comment:
My main issue with inline styles is that they are very hard to override, and AFAIK impossible to override without a `!important`.

Reply:
Yeah, I'm not a fan either, but it is definitely a pattern (or anti-pattern?) in use, at least in the learning MFE [1] (including the `flexBasis` property).
That said, we don't have a strong community theming solution for these, so it'd be nice if we can make that an easier bridge to cross when we get to it.

@davidjoy Thoughts?

- [1] https://github.com/edx/frontend-app-learning/search?q=%22style%3D%22&unscoped_q=%22style%3D%22

Reply:
I think we should explore a CSS in JS solution in Paragon and MFEs soon. This would put theming the Javascript realm and prevent us from creating CSS bloat. Adding class names to objects that don't really need them has caused the old platform's styles to bloat and become difficult to manage.

We'll start exploring CSS in JS in Paragon in the next several months. We may add a `<ThemeProvider>` component that offers theming properties like https://emotion.sh/docs/emotion-theming

Reply:
I'm fine to leave this as-is for now.